server/api/archive.go

```diff
@@ -0,0 +1,144 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/mattermost/focalboard/server/model"
+	"github.com/mattermost/focalboard/server/services/audit"
+)
+
+func (a *API) handleArchiveExport(w http.ResponseWriter, r *http.Request) {
+	// swagger:operation GET /api/v1/workspaces/{workspaceID}/archive/export archiveExport
+	//
+	// Exports an archive of all blocks for one or more boards. If root_id is provided then
+	// only that baord will be exported, otherwise all boards in the workspace.
+	//
+	// ---
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: workspaceID
+	//   in: path
+	//   description: Workspace ID
+	//   required: true
+	//   type: string
+	// - name: root_id
```

Comment:
Should we use `board_id` instead of `root_id`, are we allowing you to export something that is not a board?, eventually the tree is going to dissapear, and also the `root_id` too, so... makes sense to just call the parameter `board_id`?

Reply:
I can change that.  It's a breaking change to the API but I guess since the file format changed it can already be considered a breaking change.

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/2201/commits/4635eb5b6b79391af0ab7f7ecc008d3a7d7704a8